client/components/premium-global-styles-upgrade-modal/index.tsx

```diff
@@ -30,7 +39,15 @@ export default function PremiumGlobalStylesUpgradeModal( {
 	const translate = useTranslate();
 	const premiumPlanProduct = useSelector( ( state ) => getProductBySlug( state, PLAN_PREMIUM ) );
 	const translations = useGlobalStylesUpgradeTranslations( { numOfSelectedGlobalStyles } );
-	const isLoading = ! premiumPlanProduct;
+	const isPremiumPlanProductLoading = ! premiumPlanProduct;
+	const pricingMeta = usePricingMetaForGridPlans( {
+		planSlugs: [ PLAN_PREMIUM ],
+		storageAddOns: null,
```

Comment:
Nitpick: I think we can omit the `storageAddOns` param. 🤔

Reply:
Nope, it's mandatory:

https://github.com/Automattic/wp-calypso/blob/17ff9dffb7d4ba28101ae2a9f70e7ed114a51526/client/my-sites/plans-grid/hooks/npm-ready/data-store/use-grid-plans.tsx#L76-L84